app/models/miq_provision_workflow.rb

```diff
@@ -6,7 +6,7 @@ def self.base_model
   def self.class_for_platform(platform)
     classy = platform.classify
 
-    if classy =~ /(.*)Infra/
```

Comment:
The capture is used in `$1` 2 lines below..

Reply:
Oh geez, nice catch!